src/components/select/Option.svelte

```diff
@@ -0,0 +1,160 @@
+{#if visible}
+  <div
+    aria-selected="{isSelected}"
+    class="{classes}"
+    data-option-value="{value}"
+    data-option-label="{label}"
+    on:click="{onClick}"
+    on:mouseenter="{() => {
+      $selectStore.activeValue = value;
+    }}">
+    <div class="ant-select-item-option-content">
+      <slot>{label}</slot>
+    </div>
+    {#if ['multiple', 'tags'].includes($selectStore.mode) && isSelected}
+      <span
+        class="ant-select-item-option-state"
+        unselectable="on"
+        aria-hidden="true"
+        style="user-select: none;">
+        <CheckOutlined />
+      </span>
+    {/if}
+  </div>
+{/if}
+
+<script>
+  import { getContext, onMount, tick } from "svelte";
+  import { CheckOutlined } from "@/components/icons";
+  import classNames from "classnames";
+  import { nanoid } from "nanoid";
+  import { CONFIG_KEY, configProvider } from "@/provider/config-provider";
+
+  const config = getContext(CONFIG_KEY) || configProvider();
+  const { getPrefixCls } = $config;
+  let prefixCls = getPrefixCls("select");
+
+  // ********************** Props **********************
+
+  // The value of the option
+  export let value;
+  // The value shown to the user
+  export let label;
+  // disabled state
+  export let disabled = false;
+
+  // ********************** /Props **********************
+
+  // Context for the whole select
+  let selectStore = getContext("selectStore");
+  // Whether this option is the currently selected one
+  let isSelected;
+  // Need to know if it's the first item selected to determine if it should be active
+  let isFirstSelected;
+  // If the item is highlighted and can be selected with enter
+  let isActive;
+  // List of classes for the wrapper
+  let classes = "";
+  // Visibility based on search
+  let visible = true;
+
+  onMount(async () => {
+    await tick();
```

Comment:
Why are we doing this here ?

Reply:
That's there to make sure that `$selectStore.selectedValue` is set before setting `$selectStore.selectedLabel`. I just tried it. If I remove it the initial label doesn't get set.